.astylerc

```diff
@@ -0,0 +1,24 @@
+# .astylerc
+# http://astyle.sourceforge.net/astyle.html
+
+--mode=c
+--style=google
+--indent=tab=4
+--pad-oper
+--unpad-paren
+--align-pointer=name
+--max-code-length=80
```

Comment:
I'd up this to 100. Also, I'd like to add:
`--max-continuation-indent=80`
so that subsequent lines are more likely to align properly. The default is 40, so we get breaks like:
```
static void _referred_variable_length_edges(const QueryGraph *q,
        TrieMap *ref_entities) {
```